integration-tests/tests/link-to-new-workspace.js

```diff
@@ -19,8 +19,9 @@ const testLinkToNewWorkspaceFn = withUserToken(async ({ testUrl, page, token })
     await fillIn(page, '//*[@placeholder="Enter a name"]', `${newWorkspaceName}`)
     await click(page, clickable({ text: 'Select a billing project' }))
     await click(page, clickable({ text: `${newWorkspaceBillingAccount}` }))
-    await click(page, clickable({ text: 'Create Workspace' }))
-    await waitForNoSpinners(page)
+  await noSpinnersAfter(page, { action: () => click(page, clickable({ textContains: 'Link to a workspace' })) })
+    // Wait for bucket access to avoid sporadic failures
+    await checkBucketAccess(page, newWorkspaceBillingAccount, newWorkspaceName)
```

Comment:
Could you move the `try` to this line - right above `checkBucketAccess`?
If I understand correctly, we want to clean up/delete the workspace, but we should only need to do that if the workspace was created correctly (line 22). After that, no matter what happens we want to clean up. 

My concern here is I want to get the correct error signal reported if the test fails, but the finally is intercepting the creation error right now and reporting all errors in the `try` block as `evaluate` errors since the deletion will fail if any part of the test fails at or before line 22.

Reply:
Oh I see what you mean! Thanks for clarifying

Reply:
I added a try/catch around deleting the workspace so the error in the try block will get thrown now